src/api/Shrooms.Presentation.Api/Controllers/EmployeeListController.cs

```diff
@@ -35,6 +36,7 @@ public EmployeeListController(IMapper mapper, IUnitOfWork unitOfWork, IPermissio
 
         [HttpGet]
         [PermissionAuthorize(Permission = BasicPermissions.EmployeeList)]
+        [CacheOutput(ServerTimeSpan = WebApiConstants.OneDay)]
```

Comment:
A bit dangerous caching. I see you added invalidation on Delete and Confirm user actions, but could be there more places where employee/user is changed? Like update actions for jobinfo, room info, etc. Just to doublecheck..